lib/list_basics.dart

```diff
@@ -106,4 +108,11 @@ extension ListBasics<E> on List<E> {
   List<E> sortedCopyBy(Comparable Function(E) sortKey) {
     return List<E>.of(this)..sortBy(sortKey);
   }
+
+  /// Removes a random element of [this] and returns it.
+  ///
+  /// Returns [null] if [this] is empty.
```

Comment:
Same here, can you include a comment about `seed`?

Reply:
Done.